druid/src/core.rs

```diff
@@ -1130,10 +1216,21 @@ impl WidgetState {
             cursor_change: CursorChange::Default,
             cursor: None,
             sub_window_hosts: Vec::new(),
+            is_explicitly_disabled_new: false,
             text_registrations: Vec::new(),
+            update_focus_chain: true,
```

Comment:
do we need to track this? Can we just always update the focus change after any `ChildrenAdded` or `DisabledChanged` event?

Reply:
Ok :)